Core/NWNXCore.cpp

```diff
@@ -253,7 +254,7 @@ void NWNXCore::InitialSetupPlugins()
 
         Plugin::CreateParams params = { services };
 
-        if (services->m_config->Get<bool>("SKIP", false))
+        if (services->m_config->Get<bool>("SKIP", !!skipAllPlugins))
```

Comment:
No need for `!!` normalization, this is a `bool` which is guaranteed to fall within the possible values of a `bool`.